recipes/r-rsclient/bld.bat

```diff
@@ -1,2 +1,2 @@
-"%R%" CMD INSTALL --build . %R_ARGS%
-IF %ERRORLEVEL% NEQ 0 exit /B 1
+"%R%" CMD INSTALL --build . %R_ARGS%
```

Comment:
Please don't modify any files outside your recipe.

Reply:
I reverted it. It was changed when I pulled the latest master and I was unable to unmodify it using git checkout --. git stash did not work either.